src/device-api/actions.ts

```diff
@@ -496,6 +510,10 @@ export const getLegacyDeviceState = async () => {
 		_.toPairs(state.local?.apps)[0]?.[1]?.services,
 	)[0]?.[1];
 
+	// Default to idle when no release has been installed yet
+	// or during local mode
```

Comment:
This makes that endpoints behave differently on local mode vs in production, that is not ideal as it means you cannot rely on these endpoints during development.